index.bs

```diff
@@ -550,6 +550,18 @@ The W3C Team</h3>
 	Within W3C, the Host institutions are governed by hosting agreements;
 	the [=Hosts=] themselves are not W3C Members.
 
+	All <dfn export>liaisons</dfn> with external organizations
+	<em class="rfc2119">must</em> be coordinated by the [=Team=]
```

Comment:
1. This defining instance doesn't actually define the term. It only defines requirements related to it.
2. Given the broad definition of liaison in W3C's context, and that the issue mentioned how such formality would not be required by the more informal variants of it, do we really want to attach all these requirements to all variants of liaison?

Reply:
The older text had more commentary about what liaisons were, but that was still not terribly constraining.

Personally, I'd be happy to drop all text about Liaisons from the process, since I don't think it's terribly constraining in the first place.

Reply:
I think it's important we engage in the usual practice of allowing a liaison representative to attend meetings as if a member of a WG.